packages/feedback-service/src/resolver.ts

```diff
@@ -1,76 +1,162 @@
 import * as _ from 'lodash';
 import { Feedback } from './schema';
 const JiraApi = require('jira-client');
+const fetch = require('node-fetch');
 import { addFeedback } from './helpers';
 
+
 export const FeedbackResolver = {
   Query: {
     listFeedback(root: any, args: any, ctx: any) {
       return Feedback.find()
         .sort({ 'timestamp': -1 })
-        .then(response => response)
-        .catch(err => err);
+        .then(response => {
+          const createdBy = response.map(res => `rhatUUID_${res.createdBy}:getUsersBy(rhatUUID:"${res.createdBy}") { _id name title uid rhatUUID isActive}`);
+          const updatedBy = response.map(res => `rhatUUID_${res.updatedBy}:getUsersBy(rhatUUID:"${res.updatedBy}") { _id name title uid rhatUUID isActive}`);
+          const users = createdBy.concat(updatedBy);
+          const formattedquery = 'query' + `{${users}}`;
+          const graphql = JSON.stringify({
+            query: formattedquery,
+            variables: {}
+          });
+          const requestOptions = {
+            method: 'POST',
+            headers: { 'Content-Type': 'application/json' },
+            body: graphql
+          };
+          const graphql_api = process.env.GRAPHQL_API;
```

Comment:
Should be using the environment variables for User Service.

```suggestion
          const graphql_api = `http://${process.env.USER_SERVICE_SERVICE_HOST}:${process.env.USER_SERVICE_SERVICE_PORT}/graphql`;
```